ziti/cmd/ziti/cmd/edge/tutorials/first-service.md

```diff
@@ -22,6 +22,8 @@ you can use the quick-start script found [here](https://github.com/openziti/ziti
 
 Let's make sure we can connect with the controller. We're going to log in with the Ziti CLI, which will then let us run the rest of our operations.
 
+NOTE: If you've used the quickstart, then your username is likely admin, as is your password.
```

Comment:
better yet - the env file contains these values and puts them into your shell if you choose to source it... so basically i will often do:
```
source $HOME/.ziti/quickstart/$(hostname)/$(hostname).env
echo $ZITI_USER
echo $ZITI_PWD
```

Reply:
Changed it to tell where the env file with username/password can be found. Even better would be to check for ZITI_USER, etc, in the env, but I'll leave that for a future iteration.